sdk/include/opentelemetry/sdk/trace/simple_processor_factory.h

```diff
@@ -17,7 +17,7 @@ namespace trace
 /**
  * Factory class for SimpleSpanProcessor.
  */
-class SimpleSpanProcessorFactory
+class OPENTELEMETRY_API SimpleSpanProcessorFactory
```

Comment:
Should we export BatcSpanProcessorFactory, TracerContextFactory and SpanExporter classes too from tracer ?